include/swift/Driver/PrettyStackTrace.h

```diff
@@ -0,0 +1,76 @@
+//===--- PrettyStackTrace.h - PrettyStackTrace for the Driver ---*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_DRIVER_PRETTYSTACKTRACE_H
+#define SWIFT_DRIVER_PRETTYSTACKTRACE_H
+
+#include "llvm/Support/PrettyStackTrace.h"
+#include "swift/Driver/Types.h"
+
+namespace swift {
+namespace driver {
+
+class Action;
+class Job;
+class CommandOutput;
+
+class PrettyStackTraceDriverAction : public llvm::PrettyStackTraceEntry {
+  const Action *TheAction;
+  const char *Description;
+
+public:
+  PrettyStackTraceDriverAction(const char *desc, const Action *A)
+      : TheAction(A), Description(desc) {}
+  virtual void print(llvm::raw_ostream &OS) const;
```

Comment:
Request: `override` is good, please use it in new code! (And then I'm not sure if the `virtual` pulls its weight.)

Reply:
Oh yeah this is just copypasta; fixed.